### PR TITLE
cloud: org sso supports automatic invitation

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by zhoubasten — APPROVED** [review]
LGTM

File: tidb-cloud/tidb-cloud-org-sso-authentication.md [hunk 1]
```diff
@@ -83,8 +83,11 @@ To enable Cloud Organization SSO, take the following steps:
     >
     > The URL cannot be changed once Cloud Organization SSO is enabled. Members in your organization will only be able to log in to TiDB Cloud using your custom URL. If you need to change the configured URL later, contact [TiDB Cloud Support](/tidb-cloud/tidb-cloud-support.md) for assistance.
 
-5. If the dialog includes a list of users to be re-invited and re-join the organization once Cloud Organization SSO is enabled, take note of the list. You will need to re-invite them after the enablement if auto-provision is disabled.
-6. Click the **I understand and confirm** check box, and then click **Enable**.
+5. Click the **I understand and confirm** check box, and then click **Enable**.
+
+    > **Note:**
+    >
+    > If the dialog includes a list of users to be re-invited and re-join for Cloud Organization SSO, TiDB Cloud will send the invitation emails to the corresponding users automatically after you click **Enable**. After receiving the invitation email, each user needs to click the link in the email to verify the identity, and the custom login page shows.
```
Comment (ran-huang) [inline tidb-cloud/tidb-cloud-org-sso-authentication.md:90]:
```suggestion
    > If the dialog includes a list of users to be re-invited and re-join for Cloud Organization SSO, TiDB Cloud will automatically send the invitation emails to those users after you enable Cloud Organization SSO. After receiving the invitation email, each user needs to click the link in the email to verify their identity, and the custom login page shows.
```

**Review by ran-huang — APPROVED** [review]
rest LGTM

**Pull request merged**